README.md

```diff
@@ -9,6 +9,12 @@ but also make certain assumptions that you may not want to impose on your module
 
 ## Release Notes
 
+### version 1.6.2
+*Release*: ???
+(Earliest compatible LabKey version 19.2)
```

Comment:
Does this change actually break compatibility with 19.1?

Reply:
Ah, no.  Just a typo there.  Thanks!